x/rewards/spec/03_ante_handlers.md

```diff
@@ -0,0 +1,26 @@
+<!--
+order: 3
+-->
+
+# AnteHandlers
+
+Section describes the module ante handlers.
+
+## MinFeeDecorator
+
+The [MinFeeDecorator](https://github.com/archway-network/archway/blob/e130d74bd456be037b4e60dea7dada5d7a8760b5/x/rewards/ante/min_cons_fee.go#L19) checks if a transaction fees are greater or equal to a calculated value.
+The handler declines the transaction if the provided fees do not match the condition:
+$$
+TxFees < TxGasLimit * MinConsensusFee
+$$
+where:
+
+* *TxFees* - transaction fees provided by a user;
+* *TxGasLimit* - transaction gas limit provided by a user;
+* *MinConsensusFee* - minimum gas unit price estimated by the module;
```

Comment:
```suggestion
$$
TxFees < TxGasLimit * MinConsensusFee
$$
where:
* *TxFees* - transaction fees provided by a user;
* *TxGasLimit* - transaction gas limit provided by a user;
* *MinConsensusFee* - minimum gas unit price estimated by the module;
```

not sure why but observing the current text in preview does not showcase the Latex Syntax highlight